cmd/buildah/commit.go

```diff
@@ -11,7 +11,8 @@ import (
 	"github.com/containers/buildah/pkg/parse"
 	"github.com/containers/buildah/util"
 	"github.com/containers/common/pkg/auth"
-	"github.com/containers/image/v5/storage"
+	"github.com/containers/image/v5/pkg/shortnames"
+	storageTransport "github.com/containers/image/v5/storage"
```

Comment:
Why are you doing this change?

Reply:
Some of our packages have the same name. In this case, `storage` could very well mean `containers/storage` and I find it sometimes hard to brain-parse code with all that ambiguity. That's why I used `storageTransport` or `dockerTransport` import aliases in `libimage` in hope to make code easier to read.

It's also the reason why I called it `libimage` and not `image`. This way, readers won't confuse it with `containers/image`.